README.md

```diff
@@ -12,6 +12,7 @@ If you want to contribute to this project, see [CONTRIBUTING.md](https://github.
 - [CBA Settings](#cba-settings)
 - [Features](#features)
 - [RT Columbia mods](#rt-columbia-mods)
+- [Cam Lao Nam borders](#Cam Lao Nam borders)
```

Comment:
This will not work, update to have same format as line 14

`- [RT Columbia mods](#rt-columbia-mods)`